### ISSUE TITLE
Trouble with {} in REPL

### ISSUE THREAD

**Issue by jchayat** [lihaoyi/Ammonite#16]
```scala
@ Range(0, 5) map {_ * 2}
Compilation Failed
Main.scala:6: Missing closing brace `}' assumed here
        def $main() = {ReplBridge.shell.shellPPrint(`res3`, "res3") + " = " + ammonite.pprint.PPrint(`res3`)}
        ^
```

**Comment by lihaoyi** [comment]
Already been filed as https://issues.scala-lang.org/browse/SI-8859, hopefully @retronym will save us. Maybe we can find some workaround in the meantime

**Comment by lihaoyi** [comment]
Fixed with 0.2.2, now that I'm using my own parser which actually works
